subsys/bluetooth/host/gatt.c

```diff
@@ -1527,11 +1550,11 @@ static u8_t notify_cb(const struct bt_gatt_attr *attr, void *user_data)
 
 		conn = bt_conn_lookup_addr_le(cfg->id, &cfg->peer);
 		if (!conn) {
-#if defined(CONFIG_BT_GATT_DYNAMIC_DB)
-			if (ccc->cfg_changed == sc_ccc_cfg_changed) {
-				sc_save(cfg, data->ind_params);
+			if (IS_ENABLED(CONFIG_BT_GATT_SERVICE_CHANGED)) {
+				if (ccc->cfg_changed == sc_ccc_cfg_changed) {
+					sc_save(cfg, data->ind_params);
```

Comment:
It looks like sc_save is inside a #if defined(CONFIG_BT_GATT_SERVICE_CHANGED) block so Im not sure we can replace like this.

Reply:
Yes, just fixed it in latest push alongside another similar place.